backend/src/ee/services/ssh-host/ssh-host-types.ts

```diff
@@ -48,3 +57,19 @@ export type TIssueSshHostHostCertDTO = {
   sshHostId: string;
   publicKey: string;
 } & Omit<TProjectPermission, "projectId">;
+
```

Comment:
(ignore placement of comment)

Somehow I expected to be able to add hosts directly to host groups from this dropdown (where it would open a modal). Thoughts on this?

<img width="1505" alt="image" src="https://github.com/user-attachments/assets/7b17a89f-9a79-43e3-ac17-16143e4c9e80" />